<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deletecontact</class>
 <widget class="QDialog" name="deletecontact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_ime">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_prezime">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>120</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_email">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>150</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_telefon">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_obrisi">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Obrisi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_odustani">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Odustani</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>25</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ime:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Prezime:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>45</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Telefon:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kategorija:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_kategorija">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>256</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Podaci kontakta za brisanje</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rodjendan">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>237</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_telefon_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>237</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Datum roÄ‘enja:</string>
   </property>
  </widget>
  <zorder>pushButton_obrisi</zorder>
  <zorder>pushButton_odustani</zorder>
  <zorder>listView</zorder>
  <zorder>label_telefon</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>label_email</zorder>
  <zorder>label_kategorija</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_ime</zorder>
  <zorder>label_prezime</zorder>
  <zorder>label_2</zorder>
  <zorder>label_6</zorder>
  <zorder>label_rodjendan</zorder>
  <zorder>label_telefon_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
